packages/fxa-content-server/app/scripts/templates/settings/account_recovery/account_recovery.mustache

```diff
@@ -50,7 +50,7 @@
             <p>{{#unsafeTranslate}}<span class="note">NOTE: </span>Your Firefox Account password encrypts your Firefox data. Only your password or recovery key can access your data. <a %(escapedLearnMoreLinkAttributes)s>Learn more</a>{{/unsafeTranslate}}</p>
 
             <div class="button-row">
-                <button class="settings-button primary-button refresh-status">{{#t}}Refresh status{{/t}}</button>
+                <button class="settings-button primary-button refresh-status" type="submit">{{#t}}Refresh status{{/t}}</button>
```

Comment:
It is a lot more code to extend the `refresh-status` style? There is no form in here that is being submit.

Reply:
Very fair. I've changed this. I actually changed the approach: not applying the light grey background to disabled blue buttons, since we already have a disabled style for the blue buttons.